### ISSUE TITLE
USB wifi adapter

### ISSUE THREAD

**Issue by nickisghosty** [bigtreetech/BIGTREETECH-OCTOPUS-Pro#29]
Can I plug a USB wifi adapter into the USB-A port instead of buying a ESP wifi module?

**Comment by PZrk73** [comment]
> Can I plug a USB wifi adapter into the USB-A port instead of buying a ESP wifi module?

Yes! But it's not a fact that it will work. It depends on firmware.
